scst-scan/create-supply-chain-workload.md

```diff
@@ -0,0 +1,120 @@
+# Create Workload from SupplyChain
+
+This topic covers how to create and apply a workload from a Tanzu Supply Chain, how to observe a workload, and how to verify the scanning performed in a workload.
+
+## <a id="overview"></a> Overview
+
+* [Create and apply workload](./create-supply-chain-workload.md#create-and-apply-workload)
+* [Observe workload](./create-supply-chain-workload.md#observe-workload)
+* [Verify workload performed scanning by checking scan results](./create-supply-chain-workload.md#verify-workload-performed-scanning-by-checking-scan-results)
+
+### <a id="create-apply-workload-from-supply-chain"></a> Create and apply workload
+
+This sections covers how to create a workload from an existing [supply chain](./create-supply-chain-with-app-scanning.md).
+
+* Use tanzu cartographer plugin to create workload from supply chain:
+```
+$ tanzu cartographer workload generate
+? Select the supply chain:   [Use arrows to move, type to filter]
+> SCANNER-supply-chain-1.0.0 (11h)
+  trivy-supply-chain-1.0.0 (3d)
+kind: Sample
+apiVersion: sample.com/v1alpha1
+metadata:
+  name: example-sample
+spec:
+  registry:
+    ...
+  scanning:
+    ...
+```
+Here the user can create a supply chain workload from the:
+* [Trivy Supply Chain](./create-supply-chain-with-app-scanning.md#create-supply-chain-with-scst---scan-20-and-trivy-supply-chain-component)
+* [Supply Chain with Custom Scanner Component](./create-supply-chain-with-app-scanning.md#create-supply-chain-with-scst---scan-20-and-the-custom-scanning-component)
+
+This will render a sample workload yaml that the user can configure and put in a `workload.yaml`.
+
+* Apply workload:
+```
+kubectl apply -f workload.yaml -n DEV-NAMESPACE
+```
+
+
+### <a id="observe-workload"></a> Observe workload
+
+* Tanzu cartographer to see workload is running:
+```
+$ tanzu cartographer workload list
+NAME                           NAMESPACE                   SUPPLY CHAIN                LATEST RUN                                  READY        AGE   MESSAGE                                     PROGRESS
+Sample/golang-app-robin-test   grype-app-scanning-catalog  sample-supply-chain-1.0.0   SampleRun/golang-app-robin-test-run-ccmrq   Running      2m4s  waiting for stage buildpack-build & 1 more  ##-
+```
+
+* Trace workload progress:
+
+Install kubectl tree [plugin](https://github.com/ahmetb/kubectl-tree)
```

Comment:
**Question**: A question for product - can we tell users to install 3rd party tools?

Reply:
I removed this plugin reference and added more tanzu cartographer workload commands and a reference to there docs for more info (the supply chain team still hasn't put their observing docs out yet).